### DISCUSSION TITLE
Congrats on the launch! Open source?

### DISCUSSION THREAD

**Discussion by arunsathiya** [aws/codewhisperer-command-line-discussions#2] [General]
Congrats on the launch, @brendanfalk and team! 

Just curious if the CodeWhisperer CLI code is open source somewhere? Part of the AWS CLI codebase or something?

**Comment by brendanfalk** [comment]
Hey @arunsathiya - thanks for the post! CW is not going to be open source for the moment. Our focus for the past couple of months has been relaunching the app. Open sourcing is a big decision and one we need to discuss with Amazon in more depth. Setting expectations, I think it's unlikely we will go fully open source. Hopefully will have a definitive answer to you in the next month or so (AWS is quite focused on re:Invent right now!)

FYI Our completions specs are still open source github.com/withfig/autocomplete. We are likely looking for a maintainer if you want to help?
